src/common/dialog/DCFormWrapper.svelte

```diff
@@ -9,14 +9,20 @@
   .error {
     color: red;
   }
-  input,
   table {
     position: border-box;
     width: 100%;
     font-size: 16px;
     font-family: inherit;
   }
 
+  table :global(input) {
```

Comment:
Svelte styles are scoped to their own svelte file - children components do not inherit styles.  `table :global(input)` means all inputs which are children of tables in this component - which is what we want since the input's are in children components.

https://svelte.dev/docs#component-format-style